test/Microsoft.DotNet.ApiSymbolExtensions.Tests/AssemblySymbolLoaderTests.cs

```diff
@@ -6,12 +6,15 @@
 using System.Collections.Concurrent;
 using System.Reflection;
 using Microsoft.CodeAnalysis;
+using Microsoft.DotNet.ApiSymbolExtensions.Logging;
 using Microsoft.DotNet.Cli.Utils;
 
 namespace Microsoft.DotNet.ApiSymbolExtensions.Tests
 {
     public class AssemblySymbolLoaderTests : SdkTest
     {
+        private readonly ILog _logger = new ConsoleLog(MessageImportance.High);
```

Comment:
These AssemblySymbolLoader unit tests should not use the real log implementation (it would log to the console). Please use a mock instead (probably via the Moq framework, I think we already do that in other places in our tests).

Reply:
Is SuppressibleTestLog the correct choice?

Reply:
I see it being used in other tests, but only within its own project's test files, so it's currently internal. I'd have to make it public to reuse that here.

Reply:
The tests didn't complain. I'll use that unless you disagree.

Reply:
No. The `SuppressibleTestLog` is only relevant to APICompat and shouldn't use here. You want to mock the `ILog` interface. The `Moq` framework handles this for you automatically.